### ISSUE TITLE
[enhancement] Add middleware redirect_path to published config

### ISSUE THREAD

**Issue by forby** [subfission/cas#31]
I was confused at first when trying to set `redirect_path` in the config and then realized it was actually only `cas_redirect_path` and is quite different. It might be nice to have `redirect_path` added to the cas config or documented for those using this middleware. Unless I'm missing something entirely :)

referencing `RedirectCASAuthenticated.php:26`:

``` php
if($this->cas->isAuthenticated())
{
    return redirect(config('redirect_path', 'home'));
}
```

**Comment by subfission** [comment]
So you were partially correct.  It should be "cas_redirect_path" not "redirect_path".  This just moves the user to the specified page after successful cas auth.  This is a non-breaking change in the middleware that I will adjust.  If you are using your own middleware (recommended) then use "cas_redirect_path".

**Comment by subfission** [comment]
Please update to the fixed version 040f411 or use branch-master.

**Comment by forby** [comment]
@subfission Ah, I see.  Thanks for clarifying! And great work. 👍
